smurfs/src/index.js

```diff
@@ -1,21 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './components/App';
-import thunk from 'redux-thunk';
-import logger from 'redux-logger';
-import { createStore, applyMiddleware } from 'redux';
-import { Provider } from 'react-redux';
-import /* You need some sort of reducer */ './reducers';
+import React from "react";
```

Comment:
Novina. Is there a reason with the inconsistency of this file to the others as each other file you have done has single quotes. Why change code style here?

Reply:
I was looking at previous my projects as a reference when I got stuck. This is probably due to copying and pasting the import. I'll ensure that the code style is consistent in the future but that's a good catch!